server/util.js

```diff
@@ -23,13 +26,14 @@ function getComponentsInfo(options) {
         matches = tplName.match(re);
         componentType = matches ? matches[1] : options.rootName;
         componentName = matches ? matches[2] : tplName.replace(/\.html/, '');
+        var dataForFile = getDataForFile(path.resolve(options.componentDir, tplName));
```

Comment:
This is within a loop, so the `var` should be declared outside (just like e.g. `componentName`).